### DISCUSSION TITLE
Best practice for navigation via list items and expandables?

### DISCUSSION THREAD

**Discussion by DirkHD-ops** [zauberzeug/nicegui#2768] [Q&A]
### Question

Hey guys,

I like to implement a navigation menu in a drawer. The navigation is to contain single items as well as expandable items. 

Just wondered if there is a blueprint or best practice how to build this?

Strictly speaking: Main challenge I see is how to format the selected item and in case the item is within an expandable that this is opened.
It is not a matter of how to code all this - but maybe I oversee a more simple approach.

Thx a lot
Dirk

**Comment by v479038280** [comment]
Hope for it and hope that it could be part of nicegui.  I think make it with q-list is the right approach
